__device-tests__/gutenberg-editor-sanity-test-4-visual.test.js

```diff
@@ -219,7 +219,19 @@ describe( 'Gutenberg Editor - Test Suite 4', () => {
 			await editorPage.driver.sleep( 500 );
 
 			// Visual test check
-			const screenshot = await takeScreenshot();
+			let screenshot;
+			if ( isAndroid() ) {
+				// On Android, we take the screenshot using the block list element
+				// in order to crop some pixels using the padding argument.
+				const editorScreen = await editorPage.waitForElementToBeDisplayedByXPath(
+					'//android.widget.FrameLayout[@resource-id="android:id/content"]'
+				);
```

Comment:
This element covers the editor's content area. It might be interesting to use it as the default element when taking a screenshot, especially as it excludes device UI like the status bar. Hence, we won't need to manually remove it as we do [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/9a12cfac8e7f25ffab99ef0d8c4f2d787160fbe2/__device-tests__/utils.js#L121-L126).

Reply:
> This element covers the editor's content area. It might be interesting to use it as the default element when taking a screenshot, especially as it excludes device UI like the status bar. Hence, we won't need to manually remove it as we do [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/9a12cfac8e7f25ffab99ef0d8c4f2d787160fbe2/__device-tests__/utils.js#L121-L126).

This seems like a sensible default. 💯

Reply:
I'll work on incorporating this in a separate PR, as well as explore if the same can be achieved on iOS 👍.